src/Components/Aspire.Microsoft.Azure.Cosmos/AspireAzureCosmosDBExtensions.cs

```diff
@@ -108,6 +109,19 @@ public static class AspireAzureCosmosDBExtensions
         if (serviceKey is null)
         {
             builder.Services.AddSingleton(_ => ConfigureDb());
+
+            var csBuilder = new DbConnectionStringBuilder();
+            csBuilder.ConnectionString = settings.ConnectionString;
+
+            if (csBuilder.TryGetValue("Database", out var databaseName))
```

Comment:
Is "Database" a valid connection string property? I only see these 2:

https://github.com/Azure/azure-cosmos-dotnet-v3/blob/431c6f224747e463bed0475083d5e0d7c1afaee2/Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs#L52-L53

Would CosmosDB start throwing if it finds unrecognized properties in the connection string?

cc @Pilchie @sourabh1007

Reply:
I don't think it throws, it will ignore other properties...you can run a quick test to check it out.

Reply:
I'm actually going to propose that Cosmos start accepting a connection string with segments even for AAD auth. The reason is being able to encode extra options like this.